.github/workflows/test.yml

```diff
@@ -19,8 +19,6 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: ${{ matrix.go }}
-      - name: Install dependencies 📦
-        run: go mod tidy
```

Comment:
Q: Why remove this?

Reply:
> Q: Why remove this?

It will throw a error to request update go version.

Reply:
Is there an error if we upgrade to 1.20?

Reply:
> Is there an error if we upgrade to 1.20?

Yes, the error will be:
`go: go.mod file indicates go 1.20, but maximum version supported by tidy is 1.18`

Reply:
Hence we must remove 1.18 and 1.19 for the test.yml file if we want to upgrade to 1.20. Okay, let's remove 1.18 and 1.19 if we must do it.

Reply:
Sorry for this, you have a PR to remove 1.18 and 1.19 just now.

Reply:
> Sorry for this, you have a PR to remove 1.18 and 1.19 just now.

That PR is closed, If needed, I will finished in this PR.

Reply:
Sure, let's remove 1.18 and 1.19 and add back `Install dependencies`.